### DISCUSSION TITLE
The plugin doesn't work in FireFox anymore

### DISCUSSION THREAD

**Discussion by AlecClaire** [Anarios/return-youtube-dislike#953] [General]
Hello

Since at least 14 OCT 2023 the plugin doesn't work in FireFox anymore.

**Comment by erickpdot** [comment]
He knows about it, and he mentioned that occasionally Firefox might take a few days to approve his updates.

**Reply by ddpp1** [reply]
thank you!

**Reply by Smeti1** [reply]
If the browser useragent refers to Firefox, the tool no longer works. I use Vivaldi. It always went as soon as the useragent Firefox is stuck in it is called Temoraily Unavaiilable
